pyproject.toml

```diff
@@ -37,26 +33,45 @@ max_attribute_length=1000   # ^^^
 exclude=".venv,venv"
 profile="jinja2"
 
+[tool.ruff]
+line-length = 120
+
+target-version = "py310"
 
-[tool.flake8]
-max-line-length = 120
-count = true
+[tool.ruff.lint]
+select = [
+    "E",  # pycodestyle
+    "W",  # pycodestyle
+    "F",  # pyflakes
+    "I",  # isort
+    "B",  # flake8-bugbear
+    "C90",  # mccabe cyclomatic complexity
+    "G",  # flake8-logging-format
+]
+ignore = []
+exclude = [
+    "db/migrations/versions/",
+    "venv*",
+    ".venv*",
+    "__pycache__",
+    "fund_store/config/fund_loader_config/FAB/"
```

Comment:
We prob don't need the `db/migrations/versions/` and `fund_store` exceptions here, unless we're just pre-empting the eventual merge?

Reply:
Just a lazy copy paste on my side. I'm gonna continue to be lazy and leave them, since we can rationalise in the combination...